apps/simple-server-example/src/server/rooms.ts

```diff
@@ -0,0 +1,93 @@
+import { RoomSnapshot, TLSocketRoom } from '@tldraw/sync-core'
+import { mkdir, readFile, writeFile } from 'fs/promises'
+import { join } from 'path'
+
+// eslint-disable-next-line no-console
```

Comment:
let's add an override in eslintrc to disable no-console for this whole app, then we can write console.log normally